### ISSUE TITLE
Time column displayed as metric value

### ISSUE THREAD

**Issue by subuk** [Vertamedia/clickhouse-grafana#70]
Hello,

If I update datasource plugin from 1.5.0 to 1.5.1, time column starts to be displayed as metric value:
![image](https://user-images.githubusercontent.com/702942/43579760-d3a4c69e-965b-11e8-8586-3c2ea11b3a90.png)
![image](https://user-images.githubusercontent.com/702942/43579684-978e6c46-965b-11e8-86a2-5f3a31ab19dc.png)

I think problem somewhere in the 5ff2d16.

**Comment by subuk** [comment]
But looking through the source code I found, that I can rename time column to the name from the group by clause and problem disappears, so I am not sure that this is a bug, just unexpected behaviour, maybe it should be fixed as part of #48 

![image](https://user-images.githubusercontent.com/702942/43580356-def079c4-965d-11e8-95ea-3cc9492b7672.png)

**Comment by hagen1778** [comment]
Hi @subuk 

Indeed, there is a misleading behavior. It's connected with the assumption that fields in `GROUP BY` must be present in `SELECT`. But seems like ClickHouse is too flexible and allows to do anything we want :-) 
There is no need in wrapping request, just `GROUP BY` the field in your `SELECT` - `time`:
```
SELECT
    (intDiv(toUInt32(date), 30) * 30) * 1000 as time,
    count() as count
FROM $table
WHERE
    date >= toDate($from) AND date < toDateTime($to)
GROUP BY time
ORDER BY time
```
